### ISSUE TITLE
Following env descriptions need an update (some are empty!)

### ISSUE THREAD

**Issue by mmattel** [owncloud/ocis#5073]
In one or more services, the following env vars have either no or an incomplete description:

`OCIS_GRPC_CLIENT_TLS_CACERT`
`OCIS_HTTP_TLS_ENABLED`
`PROXY_HTTPS_CACERT`

@rhafer as discussed, pls fix

**Comment by mmattel** [comment]
@dragonchaser I would be happy to have a make target we could run highlighting empty env strings... 😅

**Comment by mmattel** [comment]
@rhafer we have a `OCIS_GRPC_CLIENT_TLS_CACERT` but I am missing a `OCIS_HTTP_TLS_CACERT` though we have `OCIS_HTTP_TLS_KEY` and `OCIS_HTTP_TLS_CERTIFICATE`.
Is this intended?

**Comment by rhafer** [comment]
> * we have a `OCIS_GRPC_CLIENT_TLS_CACERT` but I am missing a `OCIS_HTTP_TLS_CACERT` though we have `OCIS_HTTP_TLS_KEY` and `OCIS_HTTP_TLS_CERTIFICATE` in the same way we have for grpc.
>       Is this intended?

Yes. Kind of. Currently the only service doing HTTP requests to other backend services is the proxy. For which the CA certifcate is configured with `PROXY_HTTPS_CACERT`. To keep backwards compatiblity I didn't touch that.

>     * for `PROXY_TLS` --> ` Use the HTTPS server instead of the HTTP server.` Maybe adding:
>       `Consider using TLS also when running a reverse proxy to secure internal traffic.`

While the hint itself seem correct. I really think that this (the short variable descriptions) is the  wrong place for giving such operational hints. 

>     * Q: do we need for proxy_tls also a `PROXY_TLS_CACERT` ? (especially when using a reverse proxy as it then behaves like an service-to-service communication?)

not sure what you mean by that. We already have a  `PROXY_HTTPS_CACERT` settting.

>     * `LDAP_INSECURE` Add `Insecure allows to use transport security, but disables certificate verification (to be used with the autogenerated self-signed certificates)`.

For LDAP_INSECURE this is a bit missleading. I would rather leave that like it is currently.

**Comment by mmattel** [comment]
We have a lot of naming convention issues causing misunderstandings. As an example, `OCIS_HTTP_TLS_KEY` would be different to me than `OCIS_HTTP_KEY` when I follow the usual namings.

> For which the CA certifcate is configured with PROXY_HTTPS_CACERT

`OCIS_HTTP_TLS_CACERT` --> `PROXY_HTTPS_CACERT` (we should consider harmonizing the naming )

> For LDAP_INSECURE this is a bit missleading.

Why is an explanation for insecure in this context misleading?

**Comment by mmattel** [comment]
In the IDP service, the description is written different...
`Allow insecure connections to the user backend like LDAP, CS3 api, …`

This should be changed then to the text you have written above!
`Disable TLS certificate validation for the LDAP connections. Do not set this in production environments.`
